src/CodeViewer/EmTest.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <CodeViewer/Em.h>
+#include <gtest/gtest.h>
+#include <qapplication.h>
+#include <qfontdatabase.h>
+
+namespace orbit_code_viewer {
```

Comment:
I haven't and I can't follow. What would be the purpose of an anonymous namespace here?

Reply:
My main concern was about the "static int argc = 0;" that you introduce in this test, but also in others. If I am not wrong, this would be visible in the complete namespace, wouldn't it?

Reply:
Ahh, got it. Thanks for the clarification.

Since it's declared `static' it has internal linkage, which is the same behaviour as an anonymous namespace.

I usually use an anonymous namespace for more than one symbol. But if it's just one symbol, I'll add `static`. Works for variables and for functions.

In this particular case, I copied the idiom from some other test Qt-based in the codebase, such that it stays consistent.

WDYT?

Reply:
Okay, that makes sense and is fine for me. Thanks for the clarification.